### PR TITLE
Update enable-modern-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a related article which talks about outlook prompting for password and does use modern auth

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3725

File: microsoft-365/admin/security-and-compliance/enable-modern-authentication.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ To disable modern authentication on a device, set the following registry keys on
    
 ## Related articles
 [Sign in to Office 2013 with a second verification method](https://support.microsoft.com/office/2b856342-170a-438e-9a4f-3c092394d3cb)
+[Outlook prompts for password and doesn't use Modern Authentication to connect to Office 365](https://docs.microsoft.com/outlook/troubleshoot/authentication/outlook-prompt-password-modern-authentication-enabled)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/security-and-compliance/enable-modern-authentication.md:56]:
```suggestion

[Outlook prompts for password and doesn't use Modern Authentication to connect to Office 365](https://docs.microsoft.com/outlook/troubleshoot/authentication/outlook-prompt-password-modern-authentication-enabled)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Coy review - minor edit.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks.

**Pull request merged**